configs/ttalps_skimmer_signalLike_semimuonic_config.py

```diff
@@ -18,6 +20,13 @@
     "nLooseMuons": (3, 9999999),
     "nLooseElectrons": (0, 0),
     # "nGoodTightBtaggedJets": (2, 9999999),  # TODO: consider tight WP and/or 2 b-tags
+
+    # The first value is whether to apply the cut, the second is the fraction of events in data with run>=319077.
+    # To measure the second number, you can use the `utils/count_hem_events.py` script.
+    "applyHEMveto": (True, 0.6294),
+
+    # First argument: should the veto be applied? Second argument: should histograms be saved?
```

Comment:
The meaning of the second argument changed with my latest PR (you can see an example comment in histogrammer config). It doesn't do anything, and the maps are just not saved anymore - it becomes more tricky to create them when it's in NanoEvent instead of Event.